### ISSUE TITLE
Error: validation failed: unable to recognize "": no matches for kind "Deployment" in version "apps/v1beta2"

### ISSUE THREAD

**Issue by engmsaleh** [presslabs/stack#88]
**What happened**:
When I try to deploy Stack using helm I got the following error
```
2020/05/19 01:11:10 Warning: Merging destination map for chart 'mysql-operator'. The destination item 'resources' is a table and ignoring the source 'resources' as it has a non-table value of: <nil>
Error: validation failed: unable to recognize "": no matches for kind "Deployment" in version "apps/v1beta2"
```

**What you expected to happen**:
Expected to deploy it correctly

**How to reproduce it (as minimally and precisely as possible)**:
```
helm install -n stack presslabs/stack --namespace presslabs-stack \
    -f https://raw.githubusercontent.com/presslabs/stack/master/presets/minikube.yaml
```
**Anything else?**:
I think it is due to the depreciation of some API versions apps/v1beta2
I'm using Kubernetes version `1.17.5-do.0` from Digital Ocean

**Comment by engmsaleh** [comment]
I have installed it as a workaround on a downgraded Kubernetes `1.15.11-do.0`

**Comment by AMecea** [comment]
This is fixed in version `v0.9.1` by upgrading the WordPress Operator dependency in this commit https://github.com/presslabs/stack/commit/06edb357387419b106f5120157c5690cf3720de9
